storey/table.py

```diff
@@ -235,11 +251,122 @@ def __getitem__(self, key):
          """
         return self._get_static_attrs(key)
 
+    async def _flush_worker(self):
+        while not self._terminated:
+            last_flush = time.monotonic() - (self._flush_interval_milli * 1000)
+            for key, item in self._attrs_cache.items():
+                if item.last_change > last_flush:
+                    await self._persist(_PersistJob(key, None, None))
+            await asyncio.sleep(self._flush_interval_milli / 1000)
```

Comment:
This is probably not exactly the semantic we want. If interval is every 2 seconds, and it took 2 seconds or more to persist, then we don't want to sleep at all. And if it took 1 second to persist, we only want to sleep one more second. So we need to use last_flush as a reference point here. I believe this is how it's done in `_Batching`.